### ISSUE TITLE
I train datasets "ADEChallengeData2016",But loss is almost the same

### ISSUE THREAD

**Issue by lily10086** [MarvinTeichmann/tensorflow-fcn#45]
I use your code to train datasets(ADEChallengeData2016),But the loss is almost the same:
epoch:   19 || batch:   92 || l: 3.161138
epoch:   19 || batch:   93 || l: 3.038326
epoch:   19 || batch:   94 || l: 3.166589
epoch:   19 || batch:   95 || l: 3.135044
epoch:   19 || batch:   96 || l: 3.399570
epoch:   19 || batch:   97 || l: 2.850001
epoch:   19 || batch:   98 || l: 3.409142
epoch:   19 || batch:   99 || l: 3.082799
epoch:   19 || batch:  100 || l: 3.092099
epoch:   19 || batch:  101 || l: 3.262898
epoch:   19 || batch:  102 || l: 3.120981
epoch:   19 || batch:  103 || l: 3.275167
epoch:   19 || batch:  104 || l: 3.187988
epoch:   19 || batch:  105 || l: 3.516507
epoch:   19 || batch:  106 || l: 2.979967
epoch:   19 || batch:  107 || l: 3.051030
epoch:   19 || batch:  108 || l: 3.491227
epoch:   19 || batch:  109 || l: 3.281879
epoch:   19 || batch:  110 || l: 3.299110
epoch:   19 || batch:  111 || l: 3.144029

My loss code is following:
loss=tf.reduce_mean((tf.nn.sparse_softmax_cross_entropy_with_logits(logits=vgg_fcn.upscore32,labels=tf.squeeze(mask,squeeze_dims=[3]),name="entropy")))
train = tf.train.AdamOptimizer(0.005).minimize(loss)
batch_size = 15

What should I do?

**Comment by GZHermit** [comment]
Wow, @lily10086 ,buddy,could you share your program about resolving the  dataset(ADEChallengeData2016) with me？Hope for your reply.Thank you very much.

**Comment by gaokun556** [comment]
dear friend:
     i use the loss function to train my  network,but i get errors.could you please tell me what should i do about the label(the size of my label is [256,432],but not[width,hight,num_class]).Please.
thank you for ever. @lily10086

**Comment by 953693623** [comment]
dear friend:
Can you tell me how to train this network? I'm new on the job and I have no idea about it.
